core/cat/memory/vector_memory.py

```diff
@@ -159,16 +159,16 @@ def create_collection(self):
         self.client.recreate_collection(
             collection_name=self.collection_name,
             vectors_config=VectorParams(
-                size=self.embedder_size, distance=Distance.COSINE),
-            #optimizers_config=OptimizersConfigDiff(memmap_threshold=20000),
-            quantization_config=ScalarQuantization(
-                scalar=ScalarQuantizationConfig(
-                    type=ScalarType.INT8,
-                    quantile=0.75,
-                    always_ram=False
+                size=self.embedder_size, distance=Distance.COSINE, on_disk=True,),
+            optimizers_config=models.OptimizersConfigDiff(
+              default_segment_number=5,
+              indexing_threshold=0,
+            ),
+            quantization_config=BinaryQuantization(
+                scalar=BinaryQuantizationConfig(
```

Comment:
`BinaryQuantization` accept `binary` instead of `scalar` keyword argument

Reply:
Sorry for the oversights, I am directly using the online editor